vendor.conf

```diff
@@ -1,11 +1,11 @@
-k8s.io/kubernetes v1.8.1 https://github.com/kubernetes/kubernetes
-k8s.io/client-go release-5.0 https://github.com/kubernetes/client-go
-k8s.io/apimachinery release-1.8 https://github.com/kubernetes/apimachinery
-k8s.io/apiserver release-1.8 https://github.com/kubernetes/apiserver
+k8s.io/kubernetes v1.9.0-alpha.2 https://github.com/kubernetes/kubernetes
+k8s.io/client-go master https://github.com/kubernetes/client-go
```

Comment:
for reference, these `master`s will be moved back to a given branch once kubernetes 1.9 is out